update-tests/src/test/java/io/zeebe/test/ContainerState.java

```diff
@@ -45,19 +46,18 @@
             });
   }
 
-  private final Network network;
-
   private ZeebeContainer broker;
   private ZeebeGatewayContainer gateway;
   private ZeebeClient client;
   private PartitionsActuatorClient partitionsActuatorClient;
   private ManagedVolume volume;
+  private Network network;
 
   private String brokerVersion;
   private String gatewayVersion;
 
-  public ContainerState(final Network network) {
-    this.network = network;
+  public ContainerState() {
+    network = Network.SHARED;
```

Comment:
Why not move the assignment to the declaration and remove the constructor altogether?

Reply:
Ah, I think I had it final for a bit, and just didn't change it after.